helloworldwebapp/cleanup_app_and_pipelines.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+bold() {
+  echo ". $(tput bold)" "$*" "$(tput sgr0)";
+}
+
+source ~/spinnaker-for-gcp/scripts/install/properties
+
+~/spinnaker-for-gcp/scripts/manage/check_project_mismatch.sh
+
+read -p ". $(tput bold)You are about to delete all resources from the helloworldwebapp application and pipelines. This step is not reversible. Do you wish to continue (Y/n)? $(tput sgr0)" yn
+case $yn in
+  [Yy]* ) ;;
+  * ) exit;;
+esac
+
+bold "Deleting CSR repository..."
+
+gcloud source repos delete spinnaker-marketplace-helloworldwebapp
+rm -rf ~/$PROJECT_ID/spinnaker-marketplace-helloworldwebapp
+
+bold "Deleting helloworldwebapp-prod and helloworldwebapp-staging Kubernetes resources..."
+
+kubectl delete -f ~/spinnaker-for-gcp/helloworldwebapp/templates/csr/config/staging/service.yaml
+kubectl delete -f ~/spinnaker-for-gcp/helloworldwebapp/templates/csr/config/prod/service.yaml
+
+bold "Deleting Cloud Build trigger..."
+
+for trigger in $(gcloud alpha builds triggers list --filter triggerTemplate.repoName=spinnaker-marketplace-helloworldwebapp --format 'get(id)'); do
```

Comment:
I get some errors here:
```
.  Deleting helloworldwebapp-prod and helloworldwebapp-staging Kubernetes resources... 
namespace "helloworldwebapp-staging" deleted
service "helloworldwebapp-service" deleted
namespace "helloworldwebapp-prod" deleted
service "helloworldwebapp-service" deleted
.  Deleting Cloud Build trigger... 
sh: 0: getcwd() failed: No such file or directory
sh: 0: getcwd() failed: No such file or directory
sh: 0: getcwd() failed: No such file or directory
sh: 0: getcwd() failed: No such file or directory
Deleted trigger [bb4ff924-b448-4cd3-8d35-1c83cdcf4187].
.  Deleting GCR images... 
sh: 0: getcwd() failed: No such file or directory
sh: 0: getcwd() failed: No such file or directory
sh: 0: getcwd() failed: No such file or directory
sh: 0: getcwd() failed: No such file or directory
Digests:
- gcr.io/rosco-oss-131/spinnaker-marketplace-helloworldwebapp@sha256:7725cce6da179f3275471e777e3c3630a6a4b9fc62dc48baf3e7660da8dbe8f6
  Associated tags:
 - 582a740
```

Reply:
Oh, these errors happen when u ran the cleanup script from the `spinnaker-marketplace-helloworldwebapp` that got deleted in the first step. Using pushd to circumvent this